backend/src/main/java/com/ambacam/transfert/requetes/AssignStatusTO.java

```diff
@@ -0,0 +1,38 @@
+package com.ambacam.transfert.requetes;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class AssignStatusTO {
```

Comment:
Regarding the comment up there, this class would become obsolete after a discussion.

Reply:
This comment becomes obsolete